### DISCUSSION TITLE
landscape not working

### DISCUSSION THREAD

**Discussion by Ghada-Jaber** [spatie/browsershot#731] [Bugs]
hello I am trying to do a pdf in landscape format but it always returned normal

this is the code

`Browsershot::html($this->renderCertificateHtml($data))->showBackground()->landscape()->savePdf('testPdf.pdf');`

any idea what is the issue

**Comment by lvismer** [comment]
@Ghada-Jaber Did you manage to solve this, I have the same issue.

**Comment by Jiizen** [comment]
@Ghada-Jaber : Yes. Same here... Someone has a solution... ? Thank you

**Comment by ChrisSantiago82** [comment]
Landscape can also be controlled with css/html. Adding this to the css solved my issue:

```
            @media print {
                @page {
                    size: landscape
                }
            }
```

**Reply by Jiizen** [reply]
@ChrisSantiago82 : True. It's a good alternative until this bug is resolved, thank you.

**Reply by medilies** [reply]
`@media` scope isn't mandatory `@page { size: landscape }`

**Comment by ChrisSantiago82** [comment]
It's not necessary a bug. If you are using a css library, like bootstrap, they use a set if printing options, it's possible that chrome takes the css settings instead. In many cases, you won't come around to setup your html for printing anyway.
